pkg/go/transformer/dsltojson.go

```diff
@@ -6,7 +6,7 @@ import (
 
```

Comment:
All changes below are due to changes to golanglint rules which match those of `openfga/openfga`